src/utils.js

```diff
@@ -351,3 +348,14 @@ export const omitKeys = obj => {
     return newObj;
   }, {});
 };
+
+export const safeExecute = (
+  callback,
+  message = 'Error in Lumigo tracer'
+) => () => {
```

Comment:
Why you return a func?

Reply:
If we won't return a function, `safeExecute(func)` will run `func()`.
But in some cases, we don't want to run `func()` now but to pass it as a callback.

Reply:
👍